packages/admin/src/form/FinalFormRangeInput.tsx

```diff
@@ -74,21 +78,21 @@ const FinalFormRangeInputComponent: React.FunctionComponent<WithStyles<typeof st
                         <InputBase
                             name={`${name}.min`}
                             inputProps={{
-                                min: min,
-                                max: fieldValue.max,
-                                value: internalMinInput,
-                                type: "number",
+                                value: internalMinInput !== undefined ? internalMinInput : "",
+                                type: "text",
```

Comment:
why did you change that to text?

Reply:
because if it is number you cant remove the value :) undefined is only in a text field possible :)

Reply:
You can remove the value of a type=text input. Please change that back.

try here: https://jsfiddle.net/nu8cf6k9/1/